### PR TITLE
Let's go

### PR DESCRIPTION
**Pull request by yjshen324**

### CODE REVIEW

File: Module.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import java.lang.Serializable;
+
+public interface Module extends Serializable {
+
+  String getName();
+  
+  @Nullable
```
Comment (robinshine) [inline Module.java:7]: Use @NotNull instead

Comment (yjshen324) [reply Module.java:7]: Are you sure?

File: Util.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+public class Util {
+  
+  public static String getMessage() {
```
Comment (robinshine) [inline Util.java:3]: hehei

**Review by robinshine — CHANGES_REQUESTED** [review]

**Review by robinshine — COMMENTED** [review]
well done

**Comment by robinshine** [comment]
good job

**Review by robinshine — APPROVED** [review]
oh my this is cool!

**Pull request merged**